docs/admin/auth/index.mdx

```diff
@@ -491,6 +491,14 @@ Example [`openidconnect` auth provider](/admin/config/site_config#openid-connect
 }
 ```
 
+### Microsoft Entra
+
+When setting up OIDC through [Microsoft Entra](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-openid-connect-code), formerly known as Azure Active Directory, the following [OpenID Connect scopes](https://learn.microsoft.com/en-us/entra/identity-platform/scopes-oidc#openid-connect-scopes) are required:
+
+- `openid`
```

Comment:
Isn’t it true for any OIDC providers that these scopes are available? Or is this really Entra specific?

Reply:
Oh right, this seems to be generic. I'm surprised this question hasn't been asked before 🤔